assets/templates/partials/fields/fieldset-radio.tmpl

```diff
@@ -0,0 +1,26 @@
+{{/* Use model.RadioFieldset for model binding */}}
+{{ if .ValidationErr.HasValidationErr }}
+{{ template "fragments/field-error-top" .ValidationErr.ErrorItem }}
+{{ end }}
+<fieldset class="ons-fieldset">
+    <legend class="ons-fieldset__legend">
+        {{- .Legend.FuncLocalise .Language -}}
+    </legend>
+    <div class="ons-radios__items">
+        {{ range .Radios }}
+        <div class="ons-radios__item ons-radios__item--no-border ons-u-fw">
+            <div class="ons-radio ons-radio--no-border">
+                {{ template "partials/inputs/input-radio" .Input }}
```

Comment:
Don't we need something to indicate which radio reveals the text input? `.ons-js-other` looks to be the js class that does the show/hide. https://service-manual.ons.gov.uk/design-system/components/radios#example.example-radios-with-visible-text-input.html

Reply:
The `show/hide` functionality is driven by css and not `js` so any js classes are superfluous. 
I did consider including the additional classes hidden by a bool check but I thought it was a waste of computation considering it has no baring on functionality

Reply:
Cool - it's always the last one that reveals it then?